src/index.ts

```diff
@@ -210,31 +210,29 @@ export const staticPlugin = async <Prefix extends string = '/prefix'>(
         }
     })
 
+    const assetsDir = assets[0] === sep ? assets : resolve() + sep + assets
```

Comment:
I've pulled the absolute path of assets dir out of the for loop for better performance and readability. We can now call replace once per file.